drivers/net/mlx5/mlx5_vdpa.c

```diff
@@ -91,10 +91,19 @@ mlx5_vdpa_get_vdpa_features(int did, uint64_t *features)
 	return 0;
 }
 
+#define MLX5_VDPA_SUPPORTED_PROTOCOL_FEATURES (0)
```

Comment:
can you put this define along w/ MLX5_VDPA_SW_MAX_VIRTQS_SUPPORTED at the beginning of the file?

Reply:
yes of course.
Anyway, I think this PR is not needed, since you will need to define this callback in your PR as well (see my comment there)